conf/simulator/house-energy_simparam.yaml

```diff
@@ -0,0 +1,33 @@
+simulator:
+  states: [
+    "Tin",
+  ]
```

Comment:
Does it make sense to model this as a POMDP with only Tin as a state? You have schedule_index as a feature through the configs, but that probably won't be sufficient for modeling Tout and Tset?

Reply:
the simplistic model that Jill got working uses constants for `Tout` and `Tset`. So i uploaded a new Inkling to mba
![image](https://user-images.githubusercontent.com/52263960/130129006-78c63dc4-6284-4aa1-ab08-3edf5370d9df.png)